strings/strings.go

```diff
@@ -156,3 +156,85 @@ func SanitizeEmail(email string) string {
 
 	return strings.TrimSpace(email)
 }
+
+func Reverse(input string) string {
```

Comment:
I would recommend to add documentation comments on top of function as also suggested by codeRabbit.
**Note:** Not all comments from codeRabbit need to be resolved unless it's something critical or is something which is standard in library.